### PR TITLE
SI update company forms

### PR DESCRIPTION
**Pull request by susiirwin**
@rdavid1099, @danbroadbent, @NateAnderson1780, @brendandillon - hey team - i am still working on this. The challenge I am facing and have not found a good answer for is in the Edit form for City and Zipcode - the return is currently the entire object versus the Name of the city and the Zip code numbers. I am currently digging in more to figure this out. 

From what I am seeing, I should be able to do something like 
```  <%= f.label :city %>
  <%= f.text_field(:city, :name) %>
```

to get the name of the city, but that is not working. Any thoughts?

### CODE REVIEW

**Comment by susiirwin** [comment]
All tests are now passing and all references to old tables are gone now.

**Comment by susiirwin** [comment]
@brendandillon @NateAnderson1780 @danbroadbent - can someone take a look and merge if all is ok?

**Comment by susiirwin** [comment]
updated the title of the PR

File: app/controllers/locations_controller.rb [hunk 1]
```diff
@@ -31,8 +49,17 @@ def location_params
...
                                         :state,
+                                        :city,
                                         :zip_code)
     end
+
+    def us_states
+      state_options
+    end
+
+    # def additional_location_params(params)
```
Comment (rvna) [inline app/controllers/locations_controller.rb]: We should probably delete this code.

File: app/models/state_helper.rb [hunk 2]
```diff
@@ -0,0 +1,59 @@
+module StateHelper
```
Comment (rvna) [inline app/models/state_helper.rb:1]: This is a lot simpler that the method of keeping states in a table; I like it.

**Pull request merged**